scripts_Home.js

```diff
@@ -0,0 +1,4 @@
+window.addEventListener('load', function () {
+    const animatedElement = document.querySelector('.animated-element');
+    animatedElement.classList.add('start-animation');
```

Comment:
nit: I don't see the start-animation class defined elsewhere in the project, though I do see the animation working. see if the animation continues to work if this script is commented out